v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/utils/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
As there are no classes inside utils package as of now, we can remove this package and can add when needed